README.md

```diff
@@ -31,10 +32,71 @@ Clone each of the repositories and run `make push-image-devel`
  - [Alloy](https://github.com/metal-toolbox/alloy/)
  - [Flasher](https://github.com/metal-toolbox/flasher/)
  - [Flipflop](https://github.com/metal-toolbox/flipflop)
- - [Fleetscheduler](https://github.com/metal-toolbox/fleet-scheduler) (OPTIONAL, enable in values.yaml fleetscheduler.enable)
 
 This will build and push the container images to the local container registry.
 
+## 1.1 Point to local services/repositories (Optional)
```

Comment:
Theres a lot of information in the README, can this be its own doc under notes and have a link on the README to it?